cmd/kava/cmd/root.go

```diff
@@ -81,6 +88,7 @@ func NewRootCmd() *cobra.Command {
 // addSubCmds registers all the sub commands used by kava.
 func addSubCmds(rootCmd *cobra.Command, encodingConfig params.EncodingConfig, defaultNodeHome string) {
 	rootCmd.AddCommand(
+		rpc.StatusCommand(),
```

Comment:
Replaced our own status command with the cosmos rpc status command - not sure why we had a custom one in the first place.

Reply:
This was to fix a mild regression in the sdk where the json keys became capitalized in the v40 upgrade: https://github.com/Kava-Labs/kava/pull/1055

I'd keep our version. Might be able to upstream it to cosmos

Reply:
can confirm that this binary now marshals the fields like `NodeInfo`, `SyncInfo`, and `ValidatorInfo`, instead of in snake_case.
took a look through some of our automation and there are a few places we pipe `kava status` to `jq` and access fields. this change breaks those.
though usually we curl the `/status` endpoint directly which this does not affect.

interestingly, it looks like the sdk's `status` command doesn't write to stdout, which breaks piping to `jq` altogether. in any case this is a breaking change we may not want to have to work through the consequences of in this release. worth adding a ticket to unvendor the status command in a future release, i think.

Reply:
Updated to a custom one with snake_case - seems like it is outputting to stdout